src/index.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
```

Comment:
- [ ] Kindly make sure you fix your linters error instead of disabling them. This is for best practices reasons and clean code